driver/src/main/scala/org/mongodb/scala/model/ValidationLevel.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mongodb.scala.model
+
+import scala.util.Try
+
+import com.mongodb.client.model.{ ValidationLevel => JValidationLevel }
+
+/**
+ * Determines how strictly MongoDB applies the validation rules to existing documents during an insert or update.
+ *
+ * @note Requires MongoDB 3.2 or greater
+ * @since 1.1
+ */
+object ValidationLevel {
```

Comment:
It's not clear to me how this works together with the Java driver's ValidationOptions class